src/api/json/catalog.json

```diff
@@ -287,6 +289,16 @@
       "url": "http://json.schemastore.org/project-1.0.0-beta8"
     },
     {
+      "name": "project-1.0.0-rc1.json",
+      "description": "ASP.NET vNext project configuration file",
+      "url": "http://json.schemastore.org/project-1.0.0-rc1"
+    },
+    {
+      "name": "project-1.0.0-rc2.json",
+      "description": "ASP.NET Core project configuration file",
```

Comment:
BTW this name doesn't match the name in the project.json file below. In that other one it's called "DotNetCore". It should really be something like `.NET Core project configuration file` or something like that.

@ajaybhargavb can we fix?